README.md

```diff
@@ -275,6 +275,12 @@ Default: `undefined`
 
 Machine architecture the package is targeted to, used to set the `--target` option.
 
+#### options.os
+Type: `String`
+Default: Operating system platform of the host machine
+
+Operating system platform the package is targeted to, used to set the `--target` option.
```

Comment:
This should probably mention that the value corresponds to the possible values of `process.platform`, if I'm reading this PR correctly.

Reply:
It'd also be nice if there was a link to whatever defined `%_target_os`.

Reply:
Good point. I've submitted #196 to fix this. Hopefully, those changes are good enough.